File: frontend/src/types/SavePersonalInformationPayload.ts
export type SavePersonalInformationPayload = {
  firstName: string;
  middleName?: string;
  lastName: string;
  nickname?: string;
  birthdate?: Date;
  gender?: string;
  locale: string;
  timeZone?: string;
  picture?: string;
  profile?: string;
  website?: string;
};

File: backend/src/Logitar.Demo.Web/Models/Account/SavePersonalInformationPayload.cs
﻿namespace Logitar.Demo.Web.Models.Account;

public record SavePersonalInformationPayload
{
  public string FirstName { get; set; } = string.Empty;
  public string? MiddleName { get; set; }
  public string LastName { get; set; } = string.Empty;
  public string? Nickname { get; set; }

  public DateTime? Birthdate { get; set; }
  public string? Gender { get; set; }

  public string Locale { get; set; } = string.Empty;
  public string? TimeZone { get; set; }

  public string? Picture { get; set; }
  public string? Profile { get; set; }
  public string? Website { get; set; }
}
